### PR TITLE
Add CLI Wrapper around the bakery CLI

### PR DESCRIPTION
**Pull request by philschatz**
The bakery CLI operations expect different arguments for different steps and require things like collection ids and server names and a path to recipes for some steps.

This wraps the bakery CLI so that the commands are simpler:

```
poet-cli -fetch physics
poet-cli -assemble -bake physics
poet-cli physics                    # Do all the steps for the physics book
poet-cli --skip-existing physics    # Skip any steps that have already ran
```

### CODE REVIEW

**Comment by brittany-johnson** [comment]
Hello! 

I'm running into this error:

```
/Users/bj20/openstax/output-producer-service/bakery/src/cli/execute.js:86
    ...maybeTag
    ^^^
SyntaxError: Unexpected token ...
    at Object.exports.runInThisContext (vm.js:76:16)
    at Module._compile (module.js:542:28)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.runMain (module.js:604:10)
    at run (bootstrap_node.js:394:7)
    at startup (bootstrap_node.js:149:9)
    at bootstrap_node.js:509:3
./poet-cli: ERROR: could not run [node ./src/cli/execute.js run fetch --data ./temp cnx.org col11995 latest]
```

After running `./poet-cli american-government`. 

I'm on the branch `cli-wrapper`
Current dir `/Users/bj20/openstax/output-producer-service/bakery`
I've run `npm install`

**Comment by philschatz** [comment]
It may be that the version of node you are using is old. Have you successfully ran the bakery cli before? If not, you may need to upgrade the version of node to 14 (the Long-term-Support version)

**Comment by brittany-johnson** [comment]
Made it a little further this time! I had to install `jq` via brew, this may need to be added to the docs?

File: bakery/poet-cli [hunk 1]
```diff
@@ -0,0 +1,290 @@
...
+                elif [[ ${cur_operation} == "${_linkextras_arg}" ]]
+                then
+                    try node ./src/cli/execute.js run "${cur_operation}" ${generic_args} "${book_col_id}" "${server}"
+                else
+                    try node ./src/cli/execute.js run "${cur_operation}" ${generic_args} "${book_col_id}"
+                fi
+
+                # Save the hash of this directory so we can tell if it has changed
+                # & therefore subsequent steps need to re-run
+                pushd "${test_dir}" && find . -type f -exec md5sum {} \; | md5sum > "${test_dir}.dir.md5" && popd
```
Comment (brittany-johnson) [inline bakery/poet-cli:133]: I'm now getting an error pointing to this line. 
`~/OpenStax/output-producer-service/bakery/temp/col11995/fetched-book ~/OpenStax/output-producer-service/bakery
./poet-cli: line 133: ./temp/col11995/fetched-book.dir.md5: No such file or directory`

```
internal/modules/cjs/loader.js:883
  throw err;
  ^

Error: Cannot find module '/Users/bj20/OpenStax/output-producer-service/bakery/temp/col11995/fetched-book/src/cli/execute.js'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:880:15)
    at Function.Module._load (internal/modules/cjs/loader.js:725:27)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:72:12)
    at internal/main/run_main_module.js:17:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}
./poet-cli: ERROR: could not run [node ./src/cli/execute.js run assemble --data ./temp col11995]
```

Comment (brittany-johnson) [reply bakery/poet-cli:133]: This was after I ran `./poet-cli american-government` 
In `/tmp` I do see `fetched-book`

Comment (brittany-johnson) [reply bakery/poet-cli:133]: But assembling does work if I run it separately: `./poet-cli -assemble american-government`

Comment (brittany-johnson) [reply bakery/poet-cli:133]: Another thing to note, the line `find: md5sum: No such file or directory` is printed to my console at least 50+ times after running both `./poet-cli american-government` and `./poet-cli -assemble american-government`

**Comment by philschatz** [comment]
Closing this in favor of https://github.com/openstax/pipeline

**Pull request closed without merging**